src/js/import.js

```diff
@@ -319,23 +336,31 @@
     return modal;
   }
 
-  function mapToHeaders(serp, serpArray){
+  function mapToHeaders(serp, serpArray, isTaxonomyLeave){
+    var leave = (isTaxonomyLeave) ? "-leave" : "";
     return serpArray.map(serpItem => {
-      var serps =
+      var divs =
```

Comment:
I think it's better to return the whole expression (and convert indentation to 4 spaces :wink: )